packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -343,6 +343,12 @@ abstract class BindingBase {
     );
   }
 
+  /// Sends an event when a service extension's state is changed. Clients should
+  /// listen for this event to stay aware of the current service extension
+  /// state. Any service extension that manages a state should call this method
+  /// on state change.
+  ///
+  /// `value` reflects the newly updated service extension value.
```

Comment:
Should this be `@protected` to allow re-use below?

Reply:
Never mind. This doesn't work as discussed offline.

Reply:
Discussed offline. Leaving as is.